### ISSUE TITLE
emptyFallthrough false positive

### ISSUE THREAD

**Issue by mihaitodor** [go-critic/go-critic#1028]
Consider the following code:

```go
func test(level, value string) {
	switch level {
	default:
		fallthrough
	case "", "INFO":
		log.Infoln(value)
	case "TRACE":
		log.Traceln(value)
	case "DEBUG":
		log.Debugln(value)
	case "WARN":
		log.Warnln(value)
	case "ERROR":
		log.Errorln(value)
	case "FATAL":
		log.Fatalln(value)
	}
}
```

This code will will trigger the `emptyFallthrough` check. I'm not sure if there's any better way to write this code.

I tested this with gocritic installed via `GO111MODULE=on go get -v -u github.com/go-critic/go-critic/cmd/gocritic@v0.5.4`.

**Comment by quasilyte** [comment]
Thank you for reporting this.
I prepared a fix, it'll probably be merged soon.
The next release will be out in a few days.

**Comment by mihaitodor** [comment]
Thank you for the quick fix!
